### PR TITLE
Merge queries

### PR DESCRIPTION
**Pull request by adsantos**
- Get rid of "context"
- The query field received an array. If we are querying with the same filter, but a different syntax, then these tests are merged into one.

### CODE REVIEW

File: spec_operators.json [hunk 1]
```diff
@@ -22,13 +22,12 @@
...
                     "currentUser": null,
-                    "context": "When asked for tasks that have 'call' in their content and are assigned the 'work' label using query 'call & @work' in input_operators.json",
-                    "query": "call & @work",
+                    "query": ["call & @work", "  call   &  @work"],
                     "testFile": "input_operators.json",
                     "hasInvalidQuery": false,
                     "assertSections": [
                         {
-                            "name": "Search for `call` & @work",
+                            "name": "Search for `call & @work'",
```
Comment (anupcowkur) [inline spec_operators.json]: This should be `Search for 'call' & @work`

File: spec_strings.json [hunk 2]
```diff
@@ -12,8 +12,7 @@
                "id":1,
                "freezeDate" : null,
                "currentUser": null,
-               "context":"When asked for tasks with string 'Clean' using 'q:Clean' in input_strings.json",
-               "query":"q:Clean",
+               "query":["q:Clean", "Clear"],
```
Comment (anupcowkur) [inline spec_strings.json]: The second query should be `Clean`

File: spec_user.json [hunk 3]
```diff
@@ -109,8 +61,7 @@
                     "id": "7",
```
Comment (anupcowkur) [inline spec_user.json]: `id` should be `4,5,6` and so on

**Comment by anupcowkur** [comment]
Not part of your PR but there's an extra indentation at https://github.com/Doist/TDFilterKitTestSources/blob/MergeQueries/spec_labels.json#L79 . Would be nice if you could fix that too :-)

File: spec_project.json [hunk 4]
```diff
@@ -93,8 +60,7 @@
                     "id": "6",
```
Comment (anupcowkur) [inline spec_project.json]: `id` should be `4,5,6` and so on here as well

**Comment by adsantos** [comment]
@anupcowkur All fixed

**Pull request merged**